irr/include/Keycodes.h

```diff
@@ -182,4 +183,29 @@ enum EKEY_CODE
 	KEY_KEY_CODES_COUNT = 0x100 // this is not a key, but the amount of keycodes there are.
 };
 
+class Keycode : public std::variant<EKEY_CODE, wchar_t> {
```

Comment:
```c
// A Keycode is either a character produced by the key or one of Irrlicht's codes (EKEY_CODE)
```
right?

Reply:
Right. I added this annotation in 9590be17f121b09c78bd9f1d28dba0603049ec29.